.github/workflows/on-pull-request-commit.yml

```diff
@@ -12,16 +12,23 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  check-vars-and-secrets:
+    name: Check vars and secrets
+    uses: ./.github/workflows/_check-vars-and-secrets.yml
+    secrets: inherit
 
   check-excluded-packages:
+    needs: [check-vars-and-secrets]
```

Comment:
This workflow does not inherit any secret, hence we don't need to wait for `check-vars-and-secrets`

Reply:
Fixed.